test/backend/db_tests.js

```diff
@@ -0,0 +1,747 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// require('ass')
```

Comment:
Since the tests for the `test/backend/*` are run by the the storage repo, it already loads up ass. I can remove this commented out require. I think there was a problem if this was left in it loaded up ass twice and didn't like it.